### DISCUSSION TITLE
Question about settings.yml

### DISCUSSION THREAD

**Discussion by caywin25** [searxng/searxng#1919] [Q&A] [answered]
Hello,

I have a stupid question.
I installed searxng by cloning the searxng repo and I make my changes in the settings.yml file.
Everything works wonderfully.

I do my updates with: git pull
But the dumb question is: how can I keep my settings.yml changes while doing updates with git pull?

I think it's impossible, but if not how?

Thanks

**Comment by return42** [comment, marked as answer]
Create a [/etc/searxng/settings.yml](https://github.com/searxng/searxng/blob/master/utils/templates/etc/searxng/settings.yml) where you customize your settings and leave the searx/settings.yml untouched  / follow the link, where you find a good template to start with.

**Reply by caywin25** [reply]
oh yes perfect it works fine.

But if ever an update is made in settings.yml, my file /etc/searxng/settings.yml will not have this modification.

**Reply by return42** [reply]
No need for .. do you see the [use_default_settings: true](https://github.com/searxng/searxng/blob/master/utils/templates/etc/searxng/settings.yml#L3) in the /etc/searxng/settings ?

Read https://docs.searxng.org/admin/engines/settings.html#use-default-settings

**Reply by caywin25** [reply]
Ah yes ok !!!
awesome.

Thank you :)
